recipe/session/sessmodels/recipeInterface.go

```diff
@@ -34,4 +34,5 @@ type RecipeInterface struct {
 	UpdateAccessTokenPayload    *func(sessionHandle string, newAccessTokenPayload map[string]interface{}, userContext supertokens.UserContext) error
 	GetAccessTokenLifeTimeMS    *func(userContext supertokens.UserContext) (uint64, error)
 	GetRefreshTokenLifeTimeMS   *func(userContext supertokens.UserContext) (uint64, error)
+	RegenerateAccessToken       *func(accessToken string, newAccessTokenPayload map[string]interface{}, userContext supertokens.UserContext, sessionHandle string) (RegenerateAccessTokenResponse, error)
```

Comment:
- Why is there a sessionHandle in the input?
- `newAccessTokenPayload` needs to be a pointer to `map[string]interface{}` since it is optional (see node types)